packages/fluent/scss/notification/_variables.scss

```diff
@@ -0,0 +1,34 @@
+@use "../core/color-system" as *;
+@use "../core/_variables.scss" as *;
+
+// Generic styles
+$info: #CEE5FF;
+$success: #DFF6DD;
+$warning: #FFF4CE;
+$error: #FDE7E9;
```

Comment:
Only use colors defined in the color system. I would suggest extending the [default `$kendo-theme-colors` map](https://github.com/telerik/kendo-themes/blob/develop/packages/fluent/scss/core/color-system/_theme.scss#L67).